src/wp-includes/comment-template.php

```diff
@@ -1076,6 +1076,18 @@ function comment_text( $comment_id = 0, $args = array() ) {
 
 	$comment_text = get_comment_text( $comment, $args );
 
+	if ( is_admin() ) {
```

Comment:
would it be more appropriate to just check if the current user can view unfiltered html? There are some admins that might not be able to view unfiltered html, in which case this might double encode characters?

```suggestion
	if ( current_user_can( 'unfiltered_html' ) ) {
```

Reply:
Yeah, that's a good catch. I'll update it. Thank you!